tests/installation/upgrade_select.pm

```diff
@@ -41,14 +41,17 @@ sub run {
     if (match_has_tag("select-for-update")) {
         send_key $cmd{next};
     }
-    assert_screen [qw(remove-repository license-agreement license-agreement-accepted)], 240;
```

Comment:
So we don't keep compatibility with previous version? I don't know if QAM is doing such migration test, but I just wanted to be sure that this is what we want.

Reply:
For HA/SAP QAM, we don't do migration test until now and it's not planned AFAIK. I don't know for SLE QAM.

Reply:
###So we don't keep compatibility with previous version? 
In fact, I set the condition as 'if (get_var('MEDIA_UPGRADE') || is_sle('<15-SP2'))' just to keep compatibility with previous version.  
Follow the design change of SLE15SP2, we move the license page after registration.  
Anyway, I will run HA/SAP migration test to verify not affect them.

Reply:
Sorry I should have check better, you're right you keep the compatibility. I need more coffee I think!